IdentityCore/src/MSIDError.m

```diff
@@ -52,20 +52,60 @@ MSIDErrorCode MSIDErrorCodeForOAuthError(NSString *oauthError, MSIDErrorCode def
 {
     if (oauthError && [oauthError caseInsensitiveCompare:@"invalid_request"] == NSOrderedSame)
     {
-        return MSIDErrorInvalidRequest;
+        return MSIDErrorServerInvalidRequest;
     }
     if (oauthError && [oauthError caseInsensitiveCompare:@"invalid_client"] == NSOrderedSame)
     {
-        return MSIDErrorInvalidClient;
+        return MSIDErrorServerInvalidClient;
     }
     if (oauthError && [oauthError caseInsensitiveCompare:@"invalid_scope"] == NSOrderedSame)
     {
-        return MSIDErrorInvalidScope;
+        return MSIDErrorServerInvalidScope;
     }
     if (oauthError && [oauthError caseInsensitiveCompare:@"invalid_grant"] == NSOrderedSame)
     {
-        return MSIDErrorInvalidGrant;
+        return MSIDErrorServerInvalidGrant;
     }
     
     return defaultCode;
 }
+
+NSDictionary* MSIDErrorDomainsAndCodes()
+{
+    return @{ MSIDErrorDomain : @[// General Errors
+                      @(MSIDErrorInternal),
+                      @(MSIDErrorInvalidInternalParameter),
+                      @(MSIDErrorInvalidDeveloperParameter),
+                      @(MSIDErrorUnsupportedFunctionality),
+                      
+                      // Cache Errors
+                      @(MSIDErrorCacheMultipleUsers),
+                      @(MSIDErrorCacheBadFormat),
+                      
+                      // Authority Validation Errors
+                      @(MSIDErrorAuthorityValidation),
+                      @(MSIDErrorAuthorityValidation),
+                      @(MSIDErrorAuthorityValidation),
```

Comment:
MSIDErrorAuthorityValidation error is triple :)

Reply:
woops, removed.